README.md

```diff
@@ -18,6 +18,7 @@ This project took what we built for the [Portland Diaper Bank in 2016](https://g
 ### Requirements
 
 - [Docker CE](https://store.docker.com/search?type=edition&offering=community)
+- Postgres
```

Comment:
I'm new to Docker so not sure Postgres is needed if you go the Docker route?
Maybe change this line to `docker-compose` instead?

Reply:
@marcheiligers Could you comment on this?

Reply:
^ (deleted the addition)